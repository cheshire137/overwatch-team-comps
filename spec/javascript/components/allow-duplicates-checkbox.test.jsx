import renderer from 'react-test-renderer'

import AllowDuplicatesCheckbox from '../../../app/assets/javascripts/components/allow-duplicates-checkbox.jsx'

describe('AllowDuplicatesCheckbox', () => {
  let component = null
  let isAllowed = false

  const props = {
    checked: isAllowed,
    onChange: allowed => { isAllowed = allowed }
  }

  beforeEach(() => {
    component = <AllowDuplicatesCheckbox {...props} />
  })

  test('matches snapshot', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
