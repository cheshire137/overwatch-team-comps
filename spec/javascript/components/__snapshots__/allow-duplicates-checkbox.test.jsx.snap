exports[`AllowDuplicatesCheckbox matches snapshot 1`] = `
<label
  className="checkbox allow-duplicates-label"
  htmlFor="allow-duplicates">
  <input
    checked={false}
    id="allow-duplicates"
    onChange={[Function]}
    type="checkbox" />
  Allow duplicate picks
</label>
`;
